vj4/model/adaptor/contest.py

```diff
@@ -112,22 +116,27 @@ def _acm_scoreboard(is_export, _, tdoc, ranked_tsdocs, udict, pdict):
                 'value': udict[tsdoc['uid']]['uname'], 'raw': udict[tsdoc['uid']]})
     row.append({'type': 'string',
                 'value': tsdoc.get('accept', 0)})
-    if is_export: row.append({'type': 'string', 'value': tsdoc.get('time', 0.0)})
+    if is_export:
+      row.append({'type': 'string', 'value': tsdoc.get('time', 0.0)})
+      row.append({'type': 'string', 'value': template_func.format_seconds(tsdoc.get('time', 0.0))})
```

Comment:
This is model.

`template`,... are for views.

Reply:
this is referencing a utility function, which have no connection with the template. you can change the name though, which doesn't change its nature.